pkg/cluster/nodes/node.go

```diff
@@ -238,6 +238,23 @@ func (n *Node) Role() (role string, err error) {
 	return role, nil
 }
 
+// IsRunning returns the status of the node
+func (n *Node) IsRunning() (status bool, err error) {
+	// retrive the IP address of the node using docker inspect
```

Comment:
is retrieving the status not the IP, right?

Reply:
That's right! Thanks.